packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,4 +1,35 @@
+function sendWebVitals() {
+  return import(/* webpackMode: "lazy-once" */ `web-vitals`).then(
```

Comment:
Second (unrelated) comment - regarding polyfill inlined later - from `web-vitals` package readme:
> The polyfill.js script adds event listeners (to track FID cross-browser), and it records initial page visibility state as well as the timestamp of the first visibility change to hidden (to improve the accuracy of CLS, FCP, LCP, and FID).
>
> In order for it to work properly, the script must be the first script added to the page, and it must run before the browser renders any content to the screen. This is why it needs to be added to the <head> of the document.
> 
> The "standard" version of the web-vitals library includes some of the same logic found in polyfill.js. To avoid duplicating that code when using the "base+polyfill" version, the web-vitals.base.js bundle does not include any polyfill logic, instead it coordinates with the code in polyfill.js, which is why the two scripts must be used together.

If we are inlining polyfill, it seems to me we use "base" version instead of "regular" (few bytes saved?)